spec/models/apartments_spec.rb

```diff
@@ -1,7 +1,93 @@
 require 'rails_helper'
 
 RSpec.describe Apartment, type: :model do
-  it("Apartment model exists") do
-  expect{Apartment.create}.to_not raise_error 
+
+  user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+  it 'can create an apartment' do
+
+    user.apartments.create(
+      street: "string",
+      city: "string",
+      state: "string",
+      manager: "string",
+      email: "string", 
+      price: "string", 
+      bedrooms: 3, 
+      bathrooms: 3, 
+      pets: "string",
+      image: "www.urlhelper.com",
+      user_id: 11212
+    )
+    apartment = Apartment.all
+    expect(apartment.length).to eq 1
+  end
+
+  it 'user cannot create an apartment without street' do
+    apartment = user.apartments.create city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs.', bedrooms: 7, bathrooms: 7, image: "http//facebook12345", user_id: Integer
```

Comment:
Instead of writing integer as the value for user_id, you will want to provide a number.  Since you will have a user created, you will want to dynamically access the id of the user.  Please reach out if you need more clarity on this.